ui/app/components/job-editor.js

```diff
@@ -144,6 +144,22 @@ export default class JobEditor extends Component {
     }
   }
 
+  get variables() {
+    function jsonToHcl(obj) {
+      const hclLines = [];
+
+      for (const key in obj) {
+        const value = obj[key];
+        const hclValue = typeof value === 'string' ? `"${value}"` : value;
```

Comment:
This is interesting! I would have expected that HCL values would already come to you wrapped in "" when strings. Wouldn't this result in `""string""` issues?

Reply:
hmmm... good point! I guess this really is determined by the API. I tested manually and that's what worked for me. Reading through the tests in the Go code, there's nothing that would tell me what shape to expect. Maybe we'll need to play around with this a bit more and adjust accordingly!